### DISCUSSION TITLE
Ideia para uso da prop "color" nos componentes

### DISCUSSION THREAD

**Discussion by vrlopess** [TiendaNube/nimbus-design-system#109] [Ideas] [answered]
Adoraria poder utilizar a `color-sys-light-neutral-interactive-hover` mas encontro apenas a `color-sys-light-neutral-interactive` na tipagem das cores na prop do componente Text. Percebi que nem todos os componentes possuem um nome de token específico também, então seria interessante incluirmos o nome da cor ao invés do nome do token? Assim poderíamos contemplar todas as [cores](https://nimbus.tiendanube.com/es-AR/documentation/tokens/colors) disponíveis. Desse jeito conseguiria usar a neutral-40 ao invés de apenas ter a neutral-60, que está disponível no componente.

<img width="644" alt="image" src="https://user-images.githubusercontent.com/87648920/224728763-568ee6fc-9dfe-43b6-8bf3-8d9e67a5fb13.png">

<img width="571" alt="image" src="https://user-images.githubusercontent.com/87648920/224728948-79aec93c-94e9-4945-a8db-f1e159984219.png">

**Comment by jcampos89** [comment]
Oi @vrlopess fizemos uma restrição de tokens por componentes para mantermos um coerência nos usos, que são atrelado a funcionalidades,  nada que não possa ser modificado no futuro porém precisamos entender melhor qual seria a finalidade.

Você pode dar mais detalhes do contexto em que seria necessário utilizar o `color-sys-light-neutral-interactive-hover` em um texto?

**Reply by vrlopess** [reply]
Sim claro! Basicamente tenho o seguinte componente:

<img width="685" alt="image" src="https://user-images.githubusercontent.com/87648920/226421678-6156b5a9-f0e7-46c3-8fa4-cc2552f1b322.png">

Ao lado temos o texto "em Loja São Paulo" cuja cor do figma é a `#5E6163`, que seria exatamente a `neutral-interactive-hover`. No momento deixei a `neutral.interactive` e o resultado ficou assim:

<img width="467" alt="image" src="https://user-images.githubusercontent.com/87648920/226422696-474d3f00-de72-4b7d-b10b-69a65d0102df.png">

**Comment by jcampos89** [comment, marked as answer]
@vrlopess acho que ocorreu um mal entendido no momento do desenho, a cor `neutral-interactive-hover` não tem contraste suficiente para fundos claros.

Para esses casos a recomendação seria usar `neutral-textDisabled`, `neutral-textHigh`, `neutral-textLow`.

Sinta-se livre para reabrir o Discussion se restar alguma dúvida.
